monty/shutil.py

```diff
@@ -104,14 +102,20 @@ def decompress_file(filepath):
         filepath (str): Path to file.
         compression (str): A compression mode. Valid options are "gz" or
             "bz2". Defaults to "gz".
+
+    Returns:
+        str: The decompressed file path.
     """
     toks = filepath.split(".")
     file_ext = toks[-1].upper()
     if file_ext in ["BZ2", "GZ", "Z"]:
-        with open(".".join(toks[0:-1]), "wb") as f_out, zopen(filepath, "rb") as f_in:
+        out_file = ".".join(toks[0:-1])
+        with zopen(filepath, "rb") as f_in, open(out_file, "wb") as f_out:
             f_out.writelines(f_in)
         os.remove(filepath)
 
+    return out_file
```

Comment:
@janosh: move this `return` inside the `if` statement so that it only returns the string if it's decompressing a file. Otherwise, have it `return None` (as before).